src/utils/resolve.ts

```diff
@@ -114,6 +119,98 @@ function isValidNewResolver(resolver: unknown): resolver is NewResolver {
   return true
 }
 
+function legacyNodeResolve(
+  resolverOptions: NodeResolverOptions,
+  context: ChildContext | RuleContext,
+  modulePath: string,
+  sourceFile: string,
+) {
+  const {
+    extensions,
+    includeCoreModules,
+    moduleDirectory,
+    paths,
+    preserveSymlinks: symlinks,
+    package: packageJson,
+    packageFilter,
+    pathFilter,
+    packageIterator,
+    ...rest
+  } = resolverOptions
+
+  const normalizedExtensions = arraify(extensions)
+
+  const modules = arraify(moduleDirectory)
+
+  const resolver = createNodeResolver({
+    extensions: normalizedExtensions,
+    builtinModules: includeCoreModules !== false,
+    modules,
+    symlinks,
+    ...rest,
+  })
+
+  const resolved = setRuleContext(context, () =>
+    resolver.resolve(modulePath, sourceFile),
+  )
+
+  if (resolved.found) {
+    return resolved
+  }
+
+  const normalizedPaths = arraify(paths)
```

Comment:
Why not just implement `paths` options in `unrs-resolver`?

Reply:
Because `unrs-resolver` is a port of `enhanced-resolve` and `paths` is discouraged to be used.

Reply:
Is it possible to slightly modify `enhanced-resolve`'s `modules` options to accomplish this? After all, `paths` is widely supported by the community (Jest, Rollup CommonJS plugin, Rollup Node Resolve Plugin, etc. And currently us, of course).

If you are trying to push `unrs-resolver` adoption rate, it'd better support `paths`.